cirq/experiments/cross_entropy_benchmarking.py

```diff
@@ -14,14 +14,39 @@
 
 from typing import (Any, Dict, Iterable, List, NamedTuple, Optional, Sequence,
                     Set, Tuple, Union)
+import dataclasses
 import numpy as np
+import scipy
 from matplotlib import pyplot as plt
 from cirq import circuits, devices, ops, protocols, sim, work
 
 CrossEntropyPair = NamedTuple('CrossEntropyPair', [('num_cycle', int),
                                                    ('xeb_fidelity', float)])
 
 
+@dataclasses.dataclass
+class DepolarizingModel:
+    """A model of depolarizing noise.
+
+    Models the fidelity of a quantum process as an exponential decay of the form
+    f = S * p^d, where f is the fidelity, S is the coefficient, p is the
```

Comment:
Generally speaking, it makes more sense to fit XEB results to A * p^d + B to account for SPAM errors which are independent of d.

Reply:
I thought the coefficient A already accounts for SPAM errors. @sboixo suggested this model to me so perhaps he can clarify.

Reply:
I guess fitting to A * p^d + B doesn't hurt, but B should be 0 in XEB. B is not 0 in RB, there is a difference there. 
As far as I can tell, we have been using the fit to A * p^d + B so far simply because we fit it as if it was RB for convenience.

Reply:
Thanks, Sergio! FTR: Found a reference for this: section IV D in [1] and formula (35).

[1]: https://arxiv.org/pdf/1910.11333.pdf